src/components/reusable/notification/notification.stories.js

```diff
@@ -0,0 +1,104 @@
+import { html } from 'lit';
+import './index';
+import { action } from '@storybook/addon-actions';
+
+import '../overflowMenu';
+
+const notificationBodyMsg =
+  'Message, this is an additional line Ipsum iMessage, Lorem Ipsum is simply dummy and typesetting industry.';
+
+const notificationTagArr = [
+  { tagStatus: 'info', tagLabel: 'Info' },
+  { tagStatus: 'warning', tagLabel: 'Warning' },
+  { tagStatus: 'success', tagLabel: 'Success' },
+  { tagStatus: 'error', tagLabel: 'Error' },
+];
+
+export default {
+  title: 'Components/Notification',
+  component: 'kyn-notification',
+  argTypes: {
+    type: {
+      options: ['normal', 'clickable'],
+      control: { type: 'select' },
+    },
+    tagStatus: {
+      options: ['default', 'info', 'warning', 'success', 'error'],
+      control: { type: 'select' },
+    },
+  },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/CQuDZEeLiuGiALvCWjAKlu/Applications---Component-Library?type=design&node-id=9370-44581&mode=design&t=LXc9LDk5mGkf8vnl-0',
+    },
+  },
+};
+
+export const Default = {
+  args: {
+    notificationTitle: 'Notification Title',
+    notificationSubtitle: '',
+    timeStamp: '2 mins ago',
+    href: '#',
+    type: 'normal',
+    tagStatus: 'default',
+    tagLabel: '',
+  },
+  render: (args) => {
+    return html`<kyn-notification
+      notificationTitle=${args.notificationTitle}
+      notificationSubtitle=${args.notificationSubtitle}
+      timeStamp=${args.timeStamp}
+      href=${args.href}
+      type=${args.type}
+      tagStatus=${args.tagStatus}
+      tagLabel=${args.tagLabel}
+      @on-notification-click=${(e) => action(e.type)(e)}
+      style="width:464px;"
+    >
+      <kyn-overflow-menu
+        slot="action-slot"
+        anchorRight
+        @click=${(e) => e.stopPropagation()}
```

Comment:
Use both preventDefault and stopPropagation for these.

Reply:
ok

Reply:
Done.